CHANGELOG/CHANGELOG-3.6.md

```diff
@@ -39,6 +39,9 @@ See [code changes](https://github.com/etcd-io/etcd/compare/v3.5.0...v3.6.0).
 - Add command to generate [shell completion](https://github.com/etcd-io/etcd/pull/13142).
 - Add `migrate` command for downgrading/upgrading etcd data dir files.
 
+### Package `clientv3`
+- Fix [aligning zap log timestamp resolution to microseconds](https://github.com/etcd-io/etcd/pull/15239)
+
```

Comment:
I'd suggest to update 3.6 changelog after you finish the change on 3.6 (use `RFC3339NanoTimeEncoder`). Note that usually main (3.6) branch changelog should only include new features. What you have done is just bug fix. Using `RFC3339NanoTimeEncoder` for main only is a new feature.

Reply:
Good point, removed momentarily until I get to that change.  @ahrtr do you have a preference if that 3.6 time encoder change is included in this changelog pull request or should it go in a separate one?

Reply:
> do you have a preference if that 3.6 time encoder change is included in this changelog pull request or should it go in a separate one?

Definitely a separate PR.